config/config_params.go

```diff
@@ -164,6 +164,7 @@ type Config struct {
 
 	IptablesBackend                    string            `config:"oneof(legacy,nft,auto);legacy"`
 	RouteRefreshInterval               time.Duration     `config:"seconds;90"`
+	InterfaceRefreshInterval           time.Duration     `config:"seconds;90"`
```

Comment:
Shall we set default to 10 seconds as before?

Reply:
I think 10 seconds was too aggressive, it was using more CPU than expected so I decided to increase it.